build.gradle

```diff
@@ -186,11 +186,10 @@ task wrapper(type: Wrapper) {
 
 ext.schedulingFunctionalTestConfiguration = {
 
-    include 'functionaltests/**'
+    include 'functionaltests/StandardTestSuite*'
 
-    if(!project.hasProperty('regression')){
-        def regression_tests = new File(project.rootDir, "/regression_tests.txt")
```

Comment:
The `regression_tests.txt` file should also be removed as part of this PR.

Reply:
agreed